libs/core/state/src/loading/loading-state.enum.ts

```diff
@@ -1,5 +1,6 @@
 export enum DaffLoadingState {
   Resolving = 'Resolving',
-  Mutating = 'Mutating',
+	Mutating = 'Mutating',
+	Adding = 'Adding',
```

Comment:
I'm not sure if there is a better way to do this, since you can't extend enums. Let me know if you guys have any ideas.

Reply:
I think you should make a new `DaffCartItemLoadingState`. Items are special and should have their own special enum.

Reply:
it wasn't quite that simple, but I added a `DaffCartItemLoadingState`